tool/speed.cc

```diff
@@ -2875,6 +2877,9 @@ bool Speed(const std::vector<std::string> &args) {
        !SpeedEvpCipherGeneric(EVP_aes_128_ctr(), "EVP-AES-128-CTR", kTLSADLen, selected) ||
        !SpeedEvpCipherGeneric(EVP_aes_192_ctr(), "EVP-AES-192-CTR", kTLSADLen, selected) ||
        !SpeedEvpCipherGeneric(EVP_aes_256_ctr(), "EVP-AES-256-CTR", kTLSADLen, selected) ||
+       !SpeedEvpCipherGeneric(EVP_aes_128_cbc(), "EVP-AES-128-CBC", kTLSADLen, selected) ||
```

Comment:
what padding is used here? ISO10126 or PKCSv1.5?

Not a concern for this change, but ACCP [supports ISO10126 for CBC](https://github.com/corretto/amazon-corretto-crypto-provider/blob/4db841b3dbcb8d841c60c6cdd666a5c07f77c06e/csrc/aes_cbc.cpp#L251), and if we could offload that padding to an AWS-LC API, it'd be a nice simplification.

Reply:
Turns out this isn't obvious what is used: [cipher.h](https://github.com/aws/aws-lc/blob/main/include/openssl/cipher.h#L199-L205) says standard padding is used. [Reading the code](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/cipher/cipher.c#L389-L402) this uses [RFC 5652/PKCS 7](https://datatracker.ietf.org/doc/html/rfc5652#section-6.3) padding (each padding byte value is the number of padding bytes).